gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/desktop/application/tasks/AbstractJPackageTask.kt

```diff
@@ -477,7 +481,8 @@ abstract class AbstractJPackageTask @Inject constructor(
         val packageVersion = packageVersion.get()!!
         plist[PlistKeys.CFBundleShortVersionString] = packageVersion
         plist[PlistKeys.LSApplicationCategoryType] = "Unknown"
-        plist[PlistKeys.CFBundleVersion] = packageVersion
+        val packageBuildVersion = packageBuildVersion.get()!!
```

Comment:
I think `packageBuildVersion.get()!!` should be replaced with `packageBuildVersion.orNull`, since the property is marked as optional (which is good, as it allows the task to be used separately from the default plugin configuration without bothering with setting `packageBuildVersion`)

Reply:
Sounds good, I did not know this package task could be used standalone, see latest commit. It now uses `packageBuildVersion.orNull` and has `packageVersion` as a fallback (for backwards compatibility and to make packageBuildVersion really optional). What do you think about this?